Sources/StickeyCollectionViewLayout/StickyCollectionViewLayout.swift

```diff
@@ -0,0 +1,178 @@
+import UIKit
+
+public class StickyCollectionViewLayout: UICollectionViewLayout {
+
+    public var preferredItemSize = CGSize(width: 150, height: 40)
```

Comment:
I would prefer this not to be fixed. I think you can just request this from the `collectionViewLayout` property.